docs/dapp/opl/README.md

```diff
@@ -11,13 +11,71 @@ The Oasis Privacy Layer contains a set of [smart contracts](https://github.com/o
 that integrate [Sapphire](../sapphire/README.mdx) into your existing and future
 Web3 applications, introducing confidentiality.
 
-Application developers on any EVM-compatible network can add confidential state
-and selective disclosures to their application through the usage of a smart
-contract deployed on OPL without migrating their application stack. OPL enabled
-smart contracts provide a default bridge for developers to pass messages
-between Sapphire and other networks.
+Application developers on any supported network can add confidential state
+and selective disclosures to their applications through the usage of a smart
+contract deployed on OPL without leaving their existing networks.
 
 For more information about OPL and to catch the latest news, please visit the
 official [page](https://oasisprotocol.org/opl).
 
 ![Oasis Privacy Layer diagram](../images/opl/privacy-layer-diagram.png)
+
+## Supported Networks
+
+### Mainnets
+
+ * Ethereum (1), BNB Chain (56), Polygon (137), Fantom (250), Avalanche (43114), Arbitrum One (42161), Moonriver (1285), Optimism (10), Aurora (1313161554), Syscoin (57), Astar (592), Ape (16350), Arbitrum Nova (42170), Milkomeda C1 (2001), zkSync Era (324), Filecoin (314), Polygon zkEVM (1101), Oasis Sapphire (23294)
```

Comment:
How does this relate to Autoswitch-able networks here https://deploy-preview-477--trusting-archimedes-14c863.netlify.app/dapp/opl/enclave#autoswitch ?

Anyway, I would write it as a list and/or at least sort it lexicographically.

Reply:
Eh, I made it lexicographic order, but it would be easier to keep it in the order displayed on the Celer IM documentation, so if they add anything new we can easily see the difference.